### PR TITLE
Add es.yml

### PR DESCRIPTION
**Pull request by NoelDiazMesa**
Please!! Add spanish castilian.

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+es:
+  devise:
+    omniauth_callbacks:
+      success: "Sesión realizada con tu cuenta de %{kind}."
```
Comment (aitbw) [inline config/locales/es.yml:4]: I'd change `realizada` for `iniciada`.

File: config/locales/es.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  spree:
+    user_was_not_valid: "El usuario no es válido."
+    add_another_service: 'Añadir otro servicio para inicio de sesión:'
+    authentications:
+      destroy: 'Método de autenticación eliminado satisfactoriamente.'
+    back_to_authentication_methods_list: "Volver a la lista de métodos de autenticación"
+    edit_social_method: 'Editando metodo de autenticación'
+    new_social_method: 'Nuevo método de autenticación'
+    no_authentication_methods_found: "No se encuentran ningún método de autenticación"
```
Comment (aitbw) [inline config/locales/es.yml:13]: I'd rephrase this, something like: `No hay métodos de autenticación disponibles.`, what do you think?

File: config/locales/es.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    one_more_step: 'Un paso más para completar tu registro desde %{kind}'
+    remove_authentication_option_confirmation: '¿Está seguro que quiere eliminar este método de autenticación?'
+    sign_into_account: 'Puedes iniciar sesión usando:'
+    sign_in_through_one_of_these_services: 'Iniciar sesión a través de uno de estos servicios:'
```
Comment (aitbw) [inline config/locales/es.yml:17]: `Iniciar` -> `Inicia`

File: config/locales/es.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    social_api_key: 'API Clave'
```
Comment (aitbw) [inline config/locales/es.yml:18]: `API Clave` -> `Llave del API` or maybe we can use `Clave del API` :thinking:

File: config/locales/es.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    social_api_secret: 'API Secreto'
```
Comment (aitbw) [inline config/locales/es.yml:19]: `API Secreto` -> `Secreto del API`

File: config/locales/es.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    social_authentication_methods: 'Métodos de autenticación'
+    social_authentication_methods_description: 'Configurar métodos de autenticación de OAuth'
+    social_provider: 'Proveedor social'
```
Comment (aitbw) [inline config/locales/es.yml:22]: I know this is the most immediate translation, but it feels so weird. Any suggestions?

**Review by aitbw — CHANGES_REQUESTED** [review]
Hola @NoelDiazMesa! Thanks a lot for this PR. I've left you some comments I'd like you to address (and discuss) before merging this, would you mind taking a look?

Also, can you rename this file to `es-ES.yml`? That's the correct naming convention for Castillian Spanish :smile_cat:

File: config/locales/es.yml (hunk 1 above)
Comment (NoelDiazMesa) [reply config/locales/es.yml:4]: Yes, it's better

File: config/locales/es.yml (hunk 2 above)
Comment (NoelDiazMesa) [reply config/locales/es.yml:13]: ok

File: config/locales/es.yml (hunk 3 above)
Comment (NoelDiazMesa) [reply config/locales/es.yml:17]: more personal, perfect.

File: config/locales/es.yml (hunk 4 above)
Comment (NoelDiazMesa) [reply config/locales/es.yml:18]: or 'Clave publica de la API'

File: config/locales/es.yml (hunk 5 above)
Comment (NoelDiazMesa) [reply config/locales/es.yml:19]: 'Clave secreta de la API'??

**Comment by NoelDiazMesa** [comment]
The name es.yml it is because solidus_i18n implemented spanish castilian like this, if i change to es-ES.yml (it is more correct), i need to change all locales in all gem to translates correctly our projects. What do you think?? Than a lot.

File: config/locales/es.yml (hunk 4 above)
Comment (aitbw) [reply config/locales/es.yml:18]: I like your suggestion better :ok_hand:

File: config/locales/es.yml (hunk 5 above)
Comment (aitbw) [reply config/locales/es.yml:19]: Ditto. :ok_hand:

**Comment by aitbw** [comment]
@NoelDiazMesa Oh, I see it know, thanks for the explanation. Let's keep it at it is, then. :smile_cat:

File: config/locales/es.yml (hunk 6 above)
Comment (NoelDiazMesa) [reply config/locales/es.yml:22]: social_provider: 'Plataforma'
